front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -66,77 +107,120 @@
-            transition: opacity 0.2s;
-            opacity: 0;
-            font-size: 0.9rem;
-            margin-right: 0.5rem;
-          }
+      align-items: center;
+      justify-content: space-between;
+      font-weight: 600;
+      font-size: 1.5rem;
+      color: var(--black100);
+      font-size: 1.125rem;
+      text-align: left;
+      line-height: 1.5rem;
+    }
 
-          &:hover {
-            background-color: var(--secondary);
-            color: var(--white);
+    .scenario-description {
+      position: relative;
+      text-wrap: wrap;
 
-            .scenario-details-modify-button-text {
-              opacity: 1;
-            }
-          }
-        }
+      .scenario-details-description {
+        color: var(--grey50);
+        font-size: 0.875rem;
+        font-weight: 400;
+        line-height: 1.25rem;
+        margin: 0.25rem 1rem 0 0;
+        word-break: break-all;
       }
 
-      .scenario-details-infra-name {
-        display: flex;
-        align-items: center;
+      .scenario-collapse-button {
+        position: absolute;
+        padding-bottom: 0.188rem;
+        padding-inline: 0.4375rem 0.125rem;
+        height: 2.125rem;
+        width: 1.5rem;
+        color: var(--grey40);
+        background-color: var(--ambiantB10);
+        border-radius: 0.313rem 0 0 0.313rem;
+        top: 0;
+        transform: translateX(1.5rem);
+        box-shadow: 0 0px 0 1px var(--black10);
+        clip-path: inset(-1px 0 -1px -1px);
+      }
 
-        img {
-          height: 1rem;
+      .update-scenario {
+        // button has to be positioned regarding the description
+        position: absolute;
+        display: none;
+        right: 1rem;
+        bottom: -3rem;
+        padding: 0.188rem 0.5rem;
+        transition: 0.2s;
+        height: 2rem;
+        width: 2rem;
+        border-radius: 0.25rem;
+        box-shadow: 0 0 0 0.094rem rgba(255, 255, 255, 1) inset;
+        background-color: var(--ambiantB10);
+        color: var(--primary60);
+        &:hover {
+          background-color: var(--white100);
+          color: var(--primary80);
         }
       }
-
-      .scenario-details-infra-error {
-        color: var(--red);
-        font-weight: 500;
-        font-size: 0.9rem;
+    }
+    .scenario-details-electrical-profile-set {
+      font-size: 0.75rem;
+      font-weight: 400;
+      line-height: 1rem;
+      margin-top: 0.375rem;
+      color: var(--black100);
+    }
+    .scenario-details-infra-name {
+      font-size: 0.75rem;
+      font-weight: 400;
+      line-height: 1rem;
+      color: var(--black100);
+      margin-top: 0.25rem;
+      display: flex;
+      align-items: center;
+      img {
+        height: 1rem;
       }
-
-      .scenario-details-description {
-        overflow: auto;
-        max-height: 4rem;
-        font-size: 0.9rem;
-        white-space: break-spaces;
+      .infra-loading-state.cached {
+        margin: 0;
+        font-size: 1.6rem;
```

Comment:
Where does this value come from ?

Also weirdly by using the circle character, it's not really round if I zoom
![Capture d’écran 2024-10-02 à 16 26 44](https://github.com/user-attachments/assets/ca69ba81-4373-4195-8b30-4b60eb23f7a0)

Maybe we should leave the span empty and add a 8px rounded green background

```
        span {
          width: 0.5rem;
          height: 0.5rem;
          background-color: var(--success30);
          border-radius: 50%;
          margin-block: 0.3125rem 0.1875rem;
        }
```

Reply:
I think it's a better idea, I'll do it.

Reply:
Done